CharlieBackend.Api.UnitTes/StudentServiceTests.cs

```diff
@@ -74,6 +74,83 @@ public async Task CreateStudentAsync()
             Assert.Equal(ErrorCode.ValidationError, alreadyAssignedResult.Error.Code);
         }
 
+        [Fact]
+        public async Task GetStudentByIdAsync_ExistingStudentId_ShouldReturnNotNullAndSuccess()
```

Comment:
Please rename to ShouldReturnStudent and verify all expected properties in the returned model.